package.json

```diff
@@ -40,11 +40,13 @@
     "@testing-library/dom": "6.2.0",
     "@testing-library/jest-dom": "4.1.0",
     "@testing-library/react": "9.1.4",
+    "@types/dom-helpers": "3.4.1",
```

Comment:
Not sure this would be best in this PR, but `dom-helpers` is currently on `v5`. This also includes the TypeScript definitions in the package so the `@types` inclusion wouldn't be necessary. I'm not sure how big the breaking change is, but it might be worth moving.

Reply:
That sounds fair. I removed `@types/dom-helpers` and added the following to our `declarations.d.ts`.

```ts
declare module 'dom-helpers/activeElement';
```

Reply:
This actually removes the default types provided by `dom-helpers`. Could you try and remove this line and see if TS can pick up those types?

Reply:
Unfortunately, TS seems to not be able to pick up those types. Maybe we look at this together this week.

Reply:
Update: @austin94 discovered that the typings are distributed with the latest major version of `dom-helpers`. We decided to update to `^5.1.0` of `dom-helpers` so that the types can be used.